sigilrecorder.lic

```diff
@@ -0,0 +1,290 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#sigilrecorder
+=end
+
+custom_require.call(%w[common common-arcana common-items common-travel drinfomon events spellmonitor equipmanager])
+
+class SigilRecorder
+	require 'yaml'
+
+	def initialize
+		#initialize arrays for supported towns, sigils, and seasons
+		@city_list = ['Crossing', 'Riverhaven', 'Shard']
+		@sigil_list = ['abolition','congruence','induction','permutation','rarefaction', 'antipode','ascension','clarification','decay','evolution','integration','metamorphosis','nurture','paradox','unity']
+		@season_list = ['summer', 'fall', 'winter', 'spring']
+
+		#pull data and settings and store the current region and season
+		@data = get_data('sigils')
+		@settings = get_settings
+		@region = get_region #parse Room.current.image for which region/city we're in
+		@season = get_season #make sure we're recording sigils to the correct season
+		
+		#generate an array of all the room numbers associated with the current room's Room.current.image value
+		region_list = region_room_list
+		#subtract known hunting spots and any other rooms in the YAML's nogo list to avoid combat and rooms with known mapping issues
+		walk_list = generate_nogo_list(region_list)
```

Comment:
❓ Is the variable `walk_list` the rooms NOT to travel to or the rooms TO travel? I'm not sure because the method is named `generate_nogo_list`

Reply:
I am renaming this "prune_walk_list" instead to have it be more consistent with what it's doing (it is also compiling the nogo list but it then returns the modified walk list sans anything in nogo).

Reply:
Did this get renamed?